ui/package.json

```diff
@@ -83,7 +83,7 @@
     "final-form": "^4.20.6",
     "final-form-arrays": "^3.0.2",
     "final-form-calculate": "^1.3.2",
-    "gtex-d3": "github:broadinstitute/gtex-viz#8d65862",
+    "gtex-d3": "github:broadinstitute/gtex-viz-beta#13510d6",
```

Comment:
since `gtex-viz-beta` is private, need to figure out if we can/how to access these in github workflows and dev and prod

Reply:
unfortunately the components in the public repo do not work with the new api, and the components that do work with v2 are in this private repo for some reason. I asked Jared about using them in production and here's the response I got: 

> Yes, it really isn’t beta. It is what we use in production for the GTEx Portal. So just a bad name for the repo. We only deploy from the master branch so we recommend that you do not deploy from a branch below master.

Reply:
> need to figure out if we can/how to access these in github workflows and dev and prod

This is a really substantive issue and I cannot review the code until we confirm that this problem has a solution. I also do not believe that we can just add code from a private repo and add it to a public repo and have that just be okay with the gtex team, this feels like a huge problem. I highly doubt they gave you access because they were cool with us releasing all of their code under the broadest possible GPL liscense